frontend/src/components/graphs/TimeLineChart.js

```diff
@@ -3,14 +3,16 @@ import React, { useState, useEffect } from 'react';
 import { useParams } from 'react-router';
 import Chart from 'react-apexcharts';
 import { props } from 'ramda';
+import { useHistory } from 'react-router-dom';
 import Loading from '../../components/Loading';
 import { useStateValue } from '../../contexts/state';
 import { colorTypes } from '../../utils/colorTypes';
 
 const TimeLineChart = () => {
     const { seriesId } = useParams();
-    const [{ amountOfBuilds }, dispatch] = useStateValue();
+    let history = useHistory();
```

Comment:
Is there a reason that history is a let and not a const? just a minor thing though :)

Reply:
This is pure accident. Will fix this.